src/java/fr/paris/lutece/plugins/forms/util/FormsConstants.java

```diff
@@ -199,6 +200,7 @@
     public static final int ORDER_NOT_SET = Integer
             .parseInt( DatastoreService.getDataValue( "forms.formResponseStep.orderNotSet", NumberUtils.INTEGER_MINUS_ONE.toString( ) ) );
     public static final int FILE_TYPE_RIB = 1;
+    public static final String DEFAULT_REGEX_EMAIL = "((http|https)://)[a-zA-Z0-9@:\\-%.\\+~#?&//=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%.\\+~#?&;//=]*)";
```

Comment:
this seems to be a http(s) URL regex rather than an email regex

Reply:
Indeed, i'll fix that